### ISSUE TITLE
del

### ISSUE THREAD

**Issue by ghost** [Sigmmma/mek#20]

**Comment by ShellyHerself** [comment]
Have you tried using "Preserve tag values" in the sound compiler window?

(I didn't write this feature. Trying to figure out what might be wrong here)

**Comment by Apocalypse612** [comment]
> 
> 
> Have you tried using "Preserve tag values" in the sound compiler window?
> 
> (I didn't write this feature. Trying to figure out what might be wrong here)

Yes, of course. This pass was witt that feature enabled, and it's also the default setting.

**Comment by ShellyHerself** [comment]
Okay, I will look into what's going on here soon

**Comment by Apocalypse612** [comment]
Thanks, I appreciate it @gbMichelle . Having this preservation feature working will make creating my xbox enhancement mod much easier.

**Comment by Apocalypse612** [comment]
I just ran another test. On top of the above issues, generate mouth data is also assigning the wrong address. It might be paired with the above, it might not. Just thought I'd mention that, but because of this, in my case swapping in this file breaks the cutscene it's paired to altogether. I've tested this on both modified and unmodified files.

**Comment by ShellyHerself** [comment]
The address is not used when a tag is in non-meta state. It's irrelevant. The problem you're facing with re-injecting the tag data might be a problem in the other tool you're using. Mozzarilla saved tags are only meant for use with the HEK, MEK, and Invader

**Comment by Apocalypse612** [comment]
I'm pretty much using all the tools you mentioned above.

 I am using Arsenic to rebuild the xbox maps though (since it's the only tool that can rebuild xbox without problems), and assembly to view the tag file data info after for changes.

**Comment by ShellyHerself** [comment]
Do these tags work on PC, but not on Xbox? Arsenic depends on a lot of internal tag values being there that tools like tool normally generate for it

**Comment by Apocalypse612** [comment]
I can't test that because the campaign maps for PC were dummied out. I did use Refinery for the initial data/tag exports, though (not tool).

**Comment by ShellyHerself** [comment]
tool is what you use to compile maps for PC. You can load campaign on PC using commands or a different UI map
